### PR TITLE
Delete multiple records via DELETE on collection (fixes RL #126)

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

**Comment by Natim** [comment]
:+1: So great thanks !

File: cliquet/resource.py [hunk 1]
```diff
@@ -429,6 +430,24 @@ def collection_post(self):
...
         return record
 
+    @resource.view(permission='readwrite')
+    def collection_delete(self):
+        settings = self.request.registry.settings
+        enabled = settings.get('cliquet.delete_collection_enabled', 'true')
+        if not native_value(enabled):
+            raise HTTPMethodNotAllowed()
+
+        self.raise_412_if_modified()
```
Comment (leplatrem) [inline cliquet/resource.py]: XXX: not tested !

**Comment by leplatrem** [comment]
@Natim @ametaireau r?

File: cliquet/resource.py (hunk 1 above)
Comment (leplatrem) [inline cliquet/resource.py]: done.

**Pull request merged**